MapboxNavigation/NavigationMapView.swift

```diff
@@ -226,6 +227,9 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     }
     
     fileprivate func commonInit() {
+        navigationCamera = NavigationCamera(self)
```

Comment:
`NavigationCamera(self)` isn't very descriptive, and misleads me into thinking that this is something functional.

How about `NavigationCamera(mapView: MGLMapView)`?